data/sql/updates/pending_db_world/rev_1609274862007339479.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609274862007339479');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` = 17886);
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` IN (-63582, -63583, -63584, -63585));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-63583, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31411, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate I\''),
+(-63584, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31412, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate II\''),
+(-63585, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31413, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate III\''),
+(-63582, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 11, 31414, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '[DND]Sunhawk Portal Controller - On Respawn - Cast \'Stabilize Sun Gate IV\'');
+
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` IN (184850, 182026);
```

Comment:
Why update 184850? You dont add a script for it

Reply:
The scrips are added on guid

Reply:
Ah